include/svs/index/ivf/clustering.h

```diff
@@ -264,6 +270,12 @@ template <typename Data, std::integral I> struct DenseCluster {
 
     size_t size() const { return data_.size(); }
 
+    // Support for dynamic operations - SimpleData already has resize()
```

Comment:
```suggestion
    // Support for dynamic operations
```

`data_type` is templated at this point, we shouldn't mention `SimpleData`

Reply:
done